From 6387fb065d5ea16c777a0aee05b22c3cc6a0f73c Mon Sep 17 00:00:00 2001
From: Stef Walter <stefw@gnome.org>
Date: Wed, 8 Aug 2012 06:06:24 +0200
Subject: [PATCH] schema: Update description for gpg-cache-method

 * Document the various method strings that can be present here

https://bugzilla.gnome.org/show_bug.cgi?id=681081
---
 schema/org.gnome.crypto.cache.gschema.xml | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/schema/org.gnome.crypto.cache.gschema.xml b/schema/org.gnome.crypto.cache.gschema.xml
index 9a431eb..4547399 100644
--- a/schema/org.gnome.crypto.cache.gschema.xml
+++ b/schema/org.gnome.crypto.cache.gschema.xml
@@ -3,7 +3,10 @@
 		<key name="gpg-cache-method" type="s">
 			<default>'session'</default>
 			<summary>Cache Method</summary>
-			<description>The method to use for caching passphrases typed into the GPG agent.</description>
+			<description>The method to use for caching passphrases typed into the GPG agent.
+			Should be one of: 'always' caches permanently, 'session' caches until session end,
+			'idle' caches until the not used for gpg-cache-ttl seconds, 'timeout' caches until
+			gpg-cache-ttl seconds.</description>
 		</key>
 		<key name="gpg-cache-ttl" type="i">
 			<default>300</default>
-- 
1.7.11.2